challenge-01/README.md

```diff
@@ -2,41 +2,47 @@
 
 ```js
 // Declarar uma variável chamada `myvar`, sem valor.
-?
+var myvar;
 
 // Após declarada, atribua o valor 10 à variável `myvar`.
-?
+myvar = 10;
 
 // Declare uma nova variável chamada `soma`, e adicione uma instrução somando os valores 15 e 8.
-?
+var soma = 15 + 8;
 
 // Atribua à variável `soma` todo o valor dela, somando 1, usando o operador de soma abreviado.
-?
+soma++;
 
 // Atribua à variável `soma` todo o valor dela, multiplicando por 3, usando o operador de multiplicação abreviado.
-?
+soma = soma *= 3;
```

Comment:
Aqui também ele já atribui à variável soma todo o valor dela multiplicado por 3, sem precisar usar o `soma = ` no início da expressão =)